fastlane/lib/fastlane/actions/zip.rb

```diff
@@ -34,8 +34,8 @@ def self.available_options
                                          UI.user_error!("Couldn't find file/folder at path '#{File.expand_path(value)}'") unless File.exist?(value)
                                        end),
           FastlaneCore::ConfigItem.new(key: :output_path,
-                                       env_name: "FL_ZIP_OUTPUT_NAME",
-                                       description: "The name of the resulting zip file",
+                                       env_name: "FL_ZIP_OUTPUT_PATH",
```

Comment:
This change makes it to be no compatibility with elder version,
Should I change this?